specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeOneSkuUsages.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
```

Comment:
>providers/Microsoft.Compute/locations/eastus [](http://example.com/codeflow?start=65&length=44)

Is this API replacing CRP's list usage API?
[https://docs.microsoft.com/en-us/rest/api/compute/usage/list#usage](https://docs.microsoft.com/en-us/rest/api/compute/usage/list#usage)

Reply:
Yes

Reply:
We plan to go for preview on 6/15. The plr team needs stable version to generate sdks 6-8 weeks timeframe. They can get started with sdk generation, then we can plan for GA.